README.rst

```diff
@@ -61,6 +61,33 @@ If we decided to use docker, we would need to run the image and publish the jupy
 Finally, we would need to copy the url shown in the docker to our browser.
 
 
+Supported protocols
+###################
+
+SyMPC supports the following protocols.
+
+.. table::
+
+    +--------------+------------------------+-----------------------+
+    | **Protocol** | **Security Guarantee** | **Number of Parties** |
+    +==============+========================+=======================+
+    | ABY3         | Semi-honest            |                     3 |
+    +--------------+------------------------+-----------------------+
+    | ABY3         | Malicious              |                     3 |
```

Comment:
MAYBE : Could we have it like
ABY3 : Semi honest , Malicious  in a single row.
same for Falcon